hack/local-up-cluster.sh

```diff
@@ -424,7 +423,7 @@ function warning_log {
 
 function start_etcd {
     echo "Starting etcd"
-    ETCD_LOGFILE=${LOG_DIR}/etcd.log
+    export ETCD_LOGFILE=${LOG_DIR}/etcd.log
```

Comment:
In ./hack/local-up-cluster.sh line 427:
    ETCD_LOGFILE=${LOG_DIR}/etcd.log
    ^----------^ SC2034: ETCD_LOGFILE appears unused. Verify use (or export if used externally).